.circleci/git_no_checkin_in_last_day.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Detects whether there has been a Git commit in the last day on this
+# branch. Returns 1 if there has been a commit, 0 if there has not.
+
+r=`git log --name-only --since="1 day ago" -n 2`
```

Comment:
Could you replace backticks for shell command invocations with `$(...)`?

Reply:
Done